theme_zap/views/snippets/s_numbers_showcase.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<template id="s_numbers_showcase" inherit_id="website.s_numbers_showcase">
+    <!-- Grid items -->
+    <xpath expr="//div[hasclass('o_grid_item')]" position="attributes">
+        <attribute name="class" add="justify-content-center" remove="justify-content-start" separator=" "/>
+    </xpath>
```

Comment:
Same here, don't forget to remove this custo if you remove the alignment option :slightly_smiling_face: